docs/Community/Repositories.md

```diff
@@ -0,0 +1,122 @@
+    * [Rules](#rules)
+- [Creating Repositories](#creating-repositories)
+- [Removing Repositories](#removing-repositories)
+  * [Grounds for removal](#grounds-for-removal)
+
+## Repository Types
+
+### Core Repositories
+
+Core repositories are considered core components of Open Enclave. They are
+utilities, tools, applications, or libraries that are expected to be present
+in nearly every Open Enclave deployment, such as components and tools
+included in official Open Enclave releases. Additionally, the openenclave.io
+website and other project-wide infrastructure will remain in the openenclave
+github organization.
+
+#### Rules
+
+   1. Must adopt the Open Enclave [Code of Conduct](Governance.md#code-of-conduct)
+      statement in their repo.
+   2. Must use the [MIT License](../../LICENSE)
+      unless otherwise approved by the [CGC](governance/README.md).
+   3. Must adopt the [DCO](Contributing.md#developer-certificate-of-origin)
+      bot automation for pull requests
+      unless otherwise approved by the [CGC](governance/README.md).
+   4. Must follow other [contributing guidelines](Contributing.md)
+      unless otherwise approved by the [CGC](governance/README.md).
+   5. Must live under `github.com/openenclave/<project-name>`.
+   6. Code repositories must be approved by
+      [SIG-Architecture](sig-architecture/README.md).
+      Non-code repositories must be approved by the
+      [CGC](governance/README.md).
+
+### Temporary forks
+
+Sometimes the Open Enclave project will want changes to other external
+repositories.  Ideally, such changes would be contributed
+directly to the upstream project.  However, in some cases it may be
+necessary to use a temporary fork for some period of time, whether for
+development, or for use while the upstream submission is being reviewed
+and iterated on.  In such cases, a fork might be done under an individual's
+github organization or the openenenclave organization.  (A fork is not
+needed at all if a branch can be created in the related project.)
+
+If the Open Enclave SDK needs to directly depend on a fork of the related
+project, it is generally better for the fork to be done under the openenclave
+organization.  This makes it clear that Open Enclave itself, rather than
+a just a random developer, depends on the fork.
+
+#### Rules
+
+   1. Must adopt the Open Enclave [Code of Conduct](Governance.md#code-of-conduct)
+      statement in their repo.
+   2. Must use the same license and copyright rules as the forked project.
+   3. Must adopt the [DCO](Contributing.md#developer-certificate-of-origin)
+      bot automation for pull requests
+      unless otherwise approved by the [CGC](governance/README.md).
+   4. Must follow other [contributing guidelines](Contributing.md)
+      unless otherwise approved by the [CGC](governance/README.md).
+   5. Must live under `github.com/openenclave/<project-name>`.
+   6. Repository must be approved by [SIG-Architecture](sig-architecture/README.md).
```

Comment:
```suggestion
The repository must:
   1. Adopt the Open Enclave [Code of Conduct](Governance.md#code-of-conduct)
      statement in their repo.
   2. Use the same license and copyright rules as the forked project.
   3. Adopt the [DCO](Contributing.md#developer-certificate-of-origin)
      bot automation for pull requests
      unless otherwise approved by the [CGC](governance/README.md).
   4. Follow the other [contributing guidelines](Contributing.md)
      unless otherwise approved by the [CGC](governance/README.md).
   5. Be created under `github.com/openenclave/<project-name>`.
   6. Be approved by [SIG-Architecture](sig-architecture/README.md).
```

Reply:
Done